### PR TITLE
Update CheckSumUtil.java to eliminate unnecessary code

### PR DESCRIPTION
**Pull request by jluis**
Eliminate inecesary code on validateChecksum

validateChecksum returns true unless  vaildationCode is empty or null as substring then  raises an exception 
This code elminates  all the inecesary calls and keeps possible side efects

### CODE REVIEW

File: verification-server-api/src/main/java/es/gob/radarcovid/verification/util/CheckSumUtil.java [hunk 1]
```diff
@@ -27,10 +27,8 @@ public static int checkSum(String input) {
     }
 
     public static boolean validateChecksum(String validationCode) {
-        // Sample code - returns if code is correct
-        int last = Character.getNumericValue(validationCode.charAt(validationCode.length() - 1));
-        int checksum = checkSum(validationCode.substring(0, validationCode.length() - 1));
-        return last == checksum;
+        String dummy = validationCode.substring(0, validationCode.length() - 1)
```
Comment (RecuencoJones) [inline verification-server-api/src/main/java/es/gob/radarcovid/verification/util/CheckSumUtil.java]: Is the assignment required if it will never be used?

Comment (jluis) [reply verification-server-api/src/main/java/es/gob/radarcovid/verification/util/CheckSumUtil.java]: It was just for keep the side effects of the possible exceptions the same, but I also got bitten by substring method that makes the first index inclusive and the last exclusive (only noticed when reading the tests as I now rarely touch java

Comment (iCesofT) [reply verification-server-api/src/main/java/es/gob/radarcovid/verification/util/CheckSumUtil.java]: Thanks. We prefer to maintain the use of checkSum method as long as if we change how it is calculated, we don't need to change other methods.

**Comment by iCesofT** [comment]
Thanks but we prefer to maintain the use of the checkSum method as long as if we change how checksum is calculated, we know exactly where we use it.

**Pull request closed without merging**